doc/mgr/nfs.rst

```diff
@@ -533,20 +533,23 @@ The NFS log level can be adjusted using `nfs cluster config set` command (see :r
 Manual Ganesha deployment
 =========================
 
-It may be possible to deploy and manage the NFS ganesha daemons manually
-instead of allowing cephadm or rook to do so.
+It may be possible to deploy and manage the NFS ganesha daemons without
+orchestration frameworks such as cephadm or rook.
 
 .. note:: Manual configuration is not tested or fully documented; your
           mileage may vary. If you make this work, please help us by
           updating this documentation.
 
-Known issues
+Limitations
 ------------
 
-* The ``mgr/nfs`` module enumerates NFS clusters via the orchestrator API; if NFS is
-  not managed by the orchestrator (e.g., cephadm or rook) then this will not work.  It
-  may be possible to create the cluster, mark the cephadm service as 'unmanaged', but this
-  is awkward and not ideal.
+If no orchestrator module is enabled for the Ceph Manager the NFS cluster
+management commands, such as those starting with ``ceph nfs cluster``, will not
+function. However, commands that manage NFS exports, like those prefixed with
+``ceph nfs export`` are expected to work as long as the necessary RADOS objects
+have already been created. The exact RADOS items required can be seen by
+reading the sources for the ``mgr/nfs`` module (see the above note).
```

Comment:
what does `the sources for the ``mgr/nfs`` module` refer to here?

Reply:
I meant that the source code of the module is the best place to figure out what the module requires. I can drop that part - but since the previous note was "being honest" about the lack of documentation for things I would continue that trend by just mentioning that the expectations can be determined by reading the code.

Reply:
I've updated this section after reading a few other parts of the ceph docs that refer to code. These are usually developer oriented docs, but this section would mainly be for people exploring unsupported "features" of ceph anyway, so I hope it works for you.

Reply:
Yeah, I think the new version is much clearer. Thanks.